README.md

```diff
@@ -1415,6 +1290,71 @@ To run Terragrunt locally, use the `go run` command:
 go run main.go plan
 ```
 
+#### Dependencies
+
+* Terragrunt uses `glide`, a vendor package management tool for golang. See the glide repo for
+  [installation instructions](https://github.com/Masterminds/glide#install).
+
+#### AWS credentials
+
+Terragrunt uses the official [AWS SDK for Go](https://aws.amazon.com/sdk-for-go/), which
+means that it will automatically load credentials using the 
+[AWS standard approach](https://aws.amazon.com/blogs/security/a-new-and-standardized-way-to-manage-credentials-in-the-aws-sdks/). See the Go SDK guide on [Specifying Credentials](https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/configuring-sdk.html) for details, but basically your options include:
+
+1. Environment Variables
+    1. `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` for credentials (and also `AWS_SESSION_TOKEN` if using [STS temporary credentials](http://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_temp.html))
+   1. `AWS_PROFILE` for the profile
+1. [AWS credentials file](https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/configuring-sdk.html#shared-credentials-file). You can set this up by running `aws configure` and filling in the details.
+1. Run Terragrunt on an EC2 instance with an IAM Role.
+
```

Comment:
I completely agree that this section is redundant. I only added it in because the same content was in the old version of the README mentioned in #202 , so I figured I would preserve it. I am totally fine with removing it.